client/components/output.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Convert output.
+ */
+import React from "react";
+import OutputPanel from "./output-panel";
+import ConvertStore from "../stores/convert";
+
+export default class Output extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ConvertStore.getState();
```

Comment:
Was using the analogue to that in Flummox. The explicit approach here was nice for my learning.

I'll switch over to the component when I implement AJAX. Thanks!